apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -530,12 +530,24 @@ type ServerDistribution string
 const (
 	ServerDistributionCassandra = ServerDistribution("cassandra")
 	ServerDistributionDse       = ServerDistribution("dse")
+	ServerDistributionHcd       = ServerDistribution("hcd")
 )
 
+func (sd *ServerDistribution) IsCassandra() bool {
+	return *sd == ServerDistributionCassandra || *sd == ServerDistributionHcd
+}
+
+func (sd *ServerDistribution) IsDse() bool {
+	return *sd == ServerDistributionDse
+}
+
 func (kc *K8ssandraCluster) DefaultNumTokens(serverVersion *semver.Version) float64 {
-	if kc.Spec.Cassandra.ServerType == ServerDistributionCassandra && serverVersion.Major() == 3 {
+	if kc.Spec.Cassandra.ServerType.IsCassandra() && serverVersion.Major() == 3 {
 		return float64(256)
 	}
+	if kc.Spec.Cassandra.ServerType.IsCassandra() {
```

Comment:
Isn't this breaking backwards compatibility? 16 is the default for OSS versions and all previous deployments would use that.

Reply:
you're right, and I shouldn't have touched this anyway here.